### ISSUE TITLE
Restrict which models will be created by Diode

### ISSUE THREAD

**Issue by llamafilm** [netboxlabs/diode#190]
### Diode version

v0.6.0

### Feature type

Change to existing functionality

### Proposed feature or enhancement

I would like Diode to create new Devices, but not create new DeviceTypes.  So when I throw it a big CSV file full of data, if a DeviceType doesn't already exist, it would be skipped.

### Use case

DeviceTypes are more difficult to create, and this needs to be done carefully according to the manufacturer specs, such as power consumption, weight, interface count and speed, power port labels, etc.  If this is done incorrectly and then Devices are created based on it, each Device would have to be fixed later.

**Comment by rboucher-me** [comment]
You would like to run Diode ingestion where a new Device would not be created if it doesn't match with an existing Device Type: is that correct? Are there other attributes and other object dependencies where you would like to have similar behaviour? (Platform to Device, comes to mind)

**Comment by llamafilm** [comment]
Yes that's right. Other dependencies might include Site and Location.
Another use case is importing MAC address tables, where I would want to create Cables only if both devices already exist (matching the switch by name and the host Interface by MAC.)
I hope this could be flexible to allow customizing the import on lots of different criteria.
